newrelic/newrelic.go

```diff
@@ -23,6 +24,7 @@ type NewRelic struct {
 	Entities   entities.Entities
 	Plugins    plugins.Plugins
 	Synthetics synthetics.Synthetics
+	GraphQL    graphql.GraphQL
```

Comment:
Possibly rename this package to `nerdgraph` given the current pattern is package by New Relic product.